lib/find-used-modules.js

```diff
@@ -25,13 +26,19 @@ function parseFile(file, dirname) {
     fs.readFile(file + (addJs ? '.js' : ''), function(err, str) {
         if (err && addJs) return parseFile(path.join(file, 'index.js')).then(resolve).catch(reject);
         if (err) return reject(err);
-        var walk = astw(str.toString());
+        var ast = parse(str.toString(), {
+          ecmaVersion: 9,
+          allowReturnOutsideFunction: true
+        });
         var children = [];
         var modules = [];
-        walk(function(node) {
+        walk.full(ast, function(node) {
           if (node.callee && node.callee.name === 'require') {
             var name = node.arguments[0].value;
             if (name === undefined) return reject(new Error('Unable to resolve '+decode(node)+' in '+displayPath));
+            if (name[0] !== '.' && name.indexOf('/') >= 0) {
```

Comment:
How does this work with `@mapbox/check-file-dependencies`? A test case to demonstrate the expected behavior would be great.

Reply:
ahhh didn't think of that, originally I was just escaping hatching this by just returning. It seems we do that with some of the other require cases.

Where would you want to draw the line for required files we want to validate vs ignore?

Reply:
Ah. GP. All npm modules don't need to be checked as we assume their dependencies are installed if they are installed.

We should add a `@org/repo` dependency to https://github.com/mapbox/check-file-dependencies/blob/master/test/fixtures/good-npm/index.js to make sure this always works.

Reply:
@mcwhittemore updated